### ISSUE TITLE
Can't update hub variable

### ISSUE THREAD

**Issue by panosdim** [jupyterhub/grafana-dashboards#2]
When I try to update the hub variable it says that it needs a value in Query.
![image](https://user-images.githubusercontent.com/10371312/113406143-a87b1600-93b3-11eb-987d-5bb988229ebc.png)

**Comment by belfhi** [comment]
adding in the values from the file `jupyterhub.jsonnet` helped:

```
     'hub',
    datasource='$PROMETHEUS_DS',
    query='label_values(kube_namespace_status_phase, namespace)',
    regex='.*-(?:staging|prod)$',
    # FIXME: Grafana needs a manual 'refresh variables' before it populates this.
    # Maybe another API call?
    current="utoronto-prod"
```
I had to adjust the regex, too

**Comment by panosdim** [comment]
I put **label_values(kube_namespace_status_phase, namespace)** in _Query_ but still not working.

**Comment by belfhi** [comment]
> I put **label_values(kube_namespace_status_phase, namespace)** in _Query_ but still not working.

did you also adjust the regex? If your namespace is not `staging` or `prod` it might not be picked up by the regex.

**Comment by panosdim** [comment]
You are right I changed the regex and now it shows my hub but even if I select nothing happens and selects again the utoronto-prod.
![image](https://user-images.githubusercontent.com/10371312/113848860-92010000-97a1-11eb-95f7-f67900d48fb1.png)
![image](https://user-images.githubusercontent.com/10371312/113848910-a04f1c00-97a1-11eb-957f-e0366a623b2c.png)
![image](https://user-images.githubusercontent.com/10371312/113848981-b2c95580-97a1-11eb-9df9-5627477406d9.png)

**Comment by panosdim** [comment]
After I change the Refresh option it now works. Thanks.

**Comment by minrk** [comment]
I think this is fixed by #12
